ui/pages/confirmations/context/send/index.tsx

```diff
@@ -56,7 +56,7 @@ export const SendContext = createContext<SendContextType>({
   updateTo: () => undefined,
   updateToResolved: () => undefined,
   updateValue: () => undefined,
-  value: undefined,
+  value: '0',
```

Comment:
Not sure if this is better. The user now needs to delete the 0 before inputing a new value?

Reply:
Good point from Pedro. Can the 0 just be a placeholder text like the "Enter or paste a valid address" above? And once the user starts typing it gets replaced by the input? @jpuri